tests/test_base36.nim

```diff
@@ -0,0 +1,121 @@
+import unittest2
+import ../stew/base36
+
+when defined(nimHasUsed): {.used.}
+
+func hexToBytes*(a: string, result: var openArray[byte]) =
```

Comment:
This usage of `result` is slightly confusing due to Nim otherwise treating `result` specially, probably better to use another variable name.